src/Microsoft.ML.Transforms/MissingValueReplacing.cs

```diff
@@ -151,13 +151,14 @@ internal static string TestType(ColumnType type)
         {
             // Item type must have an NA value that exists and is not equal to its default value.
             Func<ColumnType, string> func = TestType<int>;
-            return Utils.MarshalInvoke(func, type.ItemType.RawType, type.ItemType);
+            var itemType = type.GetItemType();
```

Comment:
Probably harmless to keep it part of the statement, but no big deal.

Reply:
I just figured it was easier to capture the value than call the same method twice...

---
In reply to: [248020618](https://github.com/dotnet/machinelearning/pull/2131#discussion_r248020618) [](ancestors = 248020618)